### PR TITLE
fix(core): change variant values to lower case

### PR DESCRIPTION
**Pull request by danielleroux**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Unit tests (`yarn test`) were run locally and passed
- [x] Visual Regression Tests (`yarn visual-regression`) were run locally and passed
- [ ] Linting (`npm lint`) was run locally and passed

## Pull request type

Please check the type of change your PR introduces:

- [x] Bug fix
- [ ] Feature
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

GitHub Issue Number: Closing #622

## What is the new behavior?

- Button variants are all starting with an lower case
- FlipTileState states all starting with an lower case

## Does this introduce a breaking change?

- [x] Yes, see BREAKING_CHANGES.md
- [ ] No

### CODE REVIEW

File: BREAKING_CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,61 @@
...
+
+```ts
+export enum FlipTileState {
+  None = 'None',
+  Info = 'Info',
+  Warning = 'Warning',
+  Alarm = 'Alarm',
+  Primary = 'Primary',
+}
+```
```
Comment (nuke-ellington) [inline BREAKING_CHANGES.md:28]: Should be the other way around (from None to none)?

**Review by nuke-ellington — CHANGES_REQUESTED** [review]

File: BREAKING_CHANGES.md (hunk 1 above)
Comment (danielleroux) [reply BREAKING_CHANGES.md:28]: Ja

Comment (danielleroux) [reply BREAKING_CHANGES.md]: Fixed :)

**Review by nuke-ellington — APPROVED** [review]
lgtm

**Pull request merged**